ranger/ext/img_display.py

```diff
@@ -805,6 +844,8 @@ def draw(self, path, start_x, start_y, width, height):
         # finish initialization if it is the first call
         if self.needs_late_init:
             self._late_init()
+        if self.use_placeholder:
+            cmds.update({'U': 1})
```

Comment:
```suggestion
            # U: 1=enable virtual placement using unicode placeholder
            cmds.update({'U': 1})
```
The past me didn't seems to like comment